script/release

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf pkg/*
+bundle exec rake build
+gem push pkg/* --host http://gems.flexmls.com
```

Comment:
You can drop the `--host http://gems.flexmls.com` here since this is about the only gem we push to rubygems.org and do not push to our local gem host